src/controller/CHIPClusters.h

```diff
@@ -72,6 +73,23 @@ class DLL_EXPORT BasicCluster : public ClusterBase
     CHIP_ERROR ReadAttributeClusterRevision(Callback::Callback<> * onCompletion);
 };
 
+class DLL_EXPORT BindingCluster : public ClusterBase
+{
+public:
+    BindingCluster() : ClusterBase(kBindingClusterId) {}
+    ~BindingCluster() {}
+
+    // Cluster Commands
+    CHIP_ERROR Bind(Callback::Callback<> * onCompletion, uint64_t nodeId, uint16_t groupId, uint8_t endpointId,
```

Comment:
should we use chip::endpointId fas a type for endpointid?

same for node and group, but unsure if we have typedefs for those (probably we should).

Reply:
I have opened #4348 with a patch for this issue. It requires adding custom types to the ZCL definition inside `src/app/zap-templates/zcl`.